fetch.bs

```diff
@@ -1001,11 +1001,34 @@ unset.
 "<code>navigate</code>", or "<code>websocket</code>". Unless stated otherwise, it is
 "<code>no-cors</code>".
 
-<p class="note no-backref">Even though the default <a for=/>request</a>
-<a for=request>mode</a> is "<code>no-cors</code>", standards are highly
-discouraged from using it for new features. It is rather unsafe. "<code>navigate</code>" and
-"<code>websocket</code>" are special values for the HTML Standard.
-[[!HTML]]
+<div class="note no-backref">
+ <dl>
+  <dt>"<code>same-origin</code>"
+  <dd>Used to ensure requests are made to same-origin URLs. <a for=/>Fetch</a> will throw an error
+  if the request is not made to a same-origin URL.
```

Comment:
`throw an error` -> `return a <a>network error</a>`

Reply:
This seems like it should be changed throughout.

Reply:
Ah ok, because it is the fetch *method* that throws by rejecting promise, not the algorithm (which just returns network error which the method can later make use of)...